docs/site/themes/template/layouts/partials/docs-right-bar.html

```diff
@@ -5,7 +5,7 @@
             {{ if (or .IsNode .IsPage) }}
                 {{ $issueBody := printf "**On Page:** [%s](%s)" .Title .Permalink | htmlEscape }}
                 {{ $issueQuery := (querify "body" $issueBody) }}
-                <li><a href="{{ $.Site.Params.github_base_url }}/issues/new?{{ $issueQuery | safeURL }}" target="_blank">Report Issues</a></li>
+                <li><a href="{{ $.Site.Params.github_base_url }}/issues/new?assignees=&labels=kind%2Fdocs%2C+triage%2Fneeds-triage&template=documentation.md&title=" target="_blank">Report Issues</a></li>
```

Comment:
I'm not sure we want to completely get rid of the docs shortcode here: this would add to the body of the GitHub page which docs page the user had clicked the "Report issue" button. So clicking the report link on page `https://tanzucommunityedition.io/docs/latest/getting-started-standalone/` would bring me to a new, blank issue with `**On Page:** [](/docs/latest/getting-started-standalone/)` populated in the issue body.

But yes, there is alot of room for improvement in this user experience. We should be using the docs template to report issues so that users automatically get the labels and tags filled in. But we should still _linking back_ to the page users were on when they clicked this link